yamls/crd.yaml

```diff
@@ -407,6 +407,19 @@ spec:
                         type: string
                     type: object
                   type: array
+                policyRoutes:
+                  items:
```

Comment:
`install.sh` and `install-pre-1.16.sh` also need be updated

Reply:
ok, I will fix latest, and would you give me some suggestions about this featue, thanks.

Reply:
It looks good to me, nice work!